satellite/console/consoleweb/consoleql/user.go

```diff
@@ -75,6 +77,9 @@ func graphqlUserInput() *graphql.InputObject {
 			FieldPassword: &graphql.InputObjectFieldConfig{
 				Type: graphql.String,
```

Comment:
Just curious on why we need to add the `PartnerID` in the `UserInput` type but not in the `User` type?

Reply:
updated